encfs/FileUtils.cpp

```diff
@@ -1016,7 +1016,14 @@ RootPtr createV6Config(EncFS_Context *ctx, const shared_ptr<EncFS_Opts> &opts) {
     keySize = 256;
     blockSize = DefaultBlockSize;
     alg = findCipherAlgorithm("AES", keySize);
-    nameIOIface = BlockNameIO::CurrentInterface();
+
+	// If case-insensitive system, opt for Block32 filename encoding 
```

Comment:
formatting here and in the block below looks inconsistent with surrounding lines